packages/app/ui/components/GroupChannelsScreenView.tsx

```diff
@@ -188,15 +187,11 @@ export const GroupChannelsScreenView = React.memo(
       ({ item }) => {
         if (isSectionHeader(item)) {
           return (
-            <Text
-              paddingHorizontal="$l"
-              paddingVertical="$xl"
-              fontSize="$s"
-              color={listSectionTitleColor}
-              onLayout={handleHeaderLayout}
-            >
-              {item.title}
-            </Text>
+            <SectionListHeader onLayout={handleHeaderLayout}>
```

Comment:
I changed this because I noticed we were using this component in ChatList but not here, and afaict we should be using it in both places.